### ISSUE TITLE
Customizing colors

### ISSUE THREAD

**Issue by garcialo** [nvdajp/focusHighlight#3]
Would it be too difficult to allow users to customize the colors?

**Comment by towebo** [comment]
I'm using NVDA in combination with Windows Magnifyer with inverted colors and therefore red on black doesn't provide the best contrast. It would be nice if one could pick the colors that fits ones needs.

**Comment by nishimotz** [comment]
The development version of focusHighlight is as follows.

https://www.dropbox.com/s/9z3ar6vu116jxzm/focusHighlight-6.1-dev.nvda-addon?dl=0

This version uses nvda.ini.

If the following entries are added, it behaves as same as the previous versions.

```
[focusHighlight]
    [[passthrough]]
        defaultMode = False
        thickness = 12
```

Note that the minimum tested version of this add-on is NVDA 2014.1, however,
this snapshot is only tested with the current version of NVDA.

The setting GUI is planned as well, but not yet created.

**Comment by nishimotz** [comment]
The default values are as follows

```
[focusHighlight]
	[[passthrough]]
		defaultMode = True
		color = 2222ff
		dashStyle = 2
		thickness = 6
	[[focus]]
		color = ff0000
		dashStyle = 0
		thickness = 6
	[[navigator]]
		color = 00ff00
		dashStyle = 3
		thickness = 4
```


The dashStyle corresponds to this

https://docs.microsoft.com/ja-jp/windows/desktop/api/gdiplusenums/ne-gdiplusenums-dashstyle

**Comment by nishimotz** [comment]
Version 6.1 is released
https://github.com/nvdajp/focusHighlight/releases/tag/6.1
